library/src/main/java/com/novoda/downloadmanager/LiteDownloadService.java

```diff
@@ -84,7 +85,7 @@ private void acquireCpuWakeLock() {
         PowerManager powerManager = (PowerManager) getSystemService(POWER_SERVICE);
         if (powerManager != null) {
             wakeLock = powerManager.newWakeLock(PowerManager.PARTIAL_WAKE_LOCK, WAKELOCK_TAG);
-            wakeLock.acquire();
+            wakeLock.acquire(TEN_MINUTES_IN_MILLIS);
```

Comment:
@zegnus ASide and lint flagged this as an issue, are there any drawbacks that you can see with adding this timeout? I think it is fine but would be good to hear your thoughts.

Reply:
Yeah we had to add it 👍